app.rb

```diff
@@ -19,14 +20,20 @@ def initializer(size, token1, token2)
   ai = AI.new(checker, token2, token1)
   player1 = Player.new(human, token1, user_interface)
   player2 = Player.new(ai, token2, user_interface)
-  game = Game.new(state, board, checker, player1, player2)
 
   trap_signals(user_interface)
 
-  while(state[:is_playing])
-    state[:is_playing] = false
+  until(state[:stop_playing])
```

Comment:
can you also access the value via `state.stop_playing`?

Reply:
Yes, thank you very much.  I was wondering about this given one of your other comments.  I was starting right at it in the documentation but missed it.

In any case, I might make this into a class so other classes don't directly change the state.